.github/workflows/ci.yml

```diff
@@ -103,5 +103,3 @@ jobs:
     with:
       upload_to_pypi: false
       upload_to_anaconda: false
-      test_extras: tests
-      test_command: pytest --pyargs asdf
```

Comment:
These should not be removed. This is how we test the wheel generated for ASDF will function properly.